bin/automateEC2

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+ACCUMULO_REPO=$1
+ACCUMULO_TESTING_REPO=$2
+MUCHOS_REPO=$3
+
+while true; do
+
+	# builds Accumulo tarball in a temporary directory
+	mkdir ~/accumulotmp && cd ~/accumulotmp
+	git clone --single-branch --branch 2.0 --depth=1 $ACCUMULO_REPO && cd accumulo
+	mvn clean package -DskipFormat -PskipQA
+	
+	# copies the tarball to the given muchos directory
+	cp ./assemble/target/*.gz $MUCHOS_REPO/conf/upload/
+	if [ $? -eq 0 ]; then
+		echo "Accumulo tarball copied to fluo-muchos"
+		rm -rf ~/accumulotmp
+	else
+		break
+		rm -rf ~/accumulotmp
```

Comment:
The line after the break is unreachable.  I think you meant to before the break.